src/frontend/src/components/ChangesList.tsx

```diff
@@ -26,7 +26,7 @@ const ChangesList: React.FC<ChangesListProps> = ({ changes }) => {
           <Link component={RouterLink} to={`${routes.CHANGE_REQUESTS}/${ic.changeRequestId}`}>
             #{ic.changeRequestId}
           </Link>
-          ]
+          ]{' '}
```

Comment:
this looks like this on my local rn 
<img width="960" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/7fb1d091-cc0f-4b51-9f6f-768a70f58d7f">
I would look at what I did on the changes tab of the projects page, copy that into the changes component and use it for both